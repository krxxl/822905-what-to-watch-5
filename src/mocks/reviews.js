export default [
  {
    id: 1,
    reviews: [
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 1
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Paula Fleri-Soler`,
        date: `December 20, 2016`,
        rating: 8.9,
        id: 2
      },
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Bill Goodykoontz`,
        date: `November 18, 2015`,
        rating: 8.9,
        id: 3
      },
    ]
  },
  {
    id: 2,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 20, 2016`,
        rating: 8.9,
        id: 4
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Bill Goodykoontz`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 5
      },
    ]
  },
  {
    id: 3,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 6
      },
    ]
  },
  {
    id: 4,
    reviews: [
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Matthew Lickona`,
        date: `November 18, 2015`,
        rating: 8.9,
        id: 7
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Paula Fleri-Soler`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 8
      },
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Bill Goodykoontz`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 9
      },
    ]
  },
  {
    id: 5,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 10
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Paula Fleri-Soler`,
        date: `December 20, 2016`,
        rating: 8.9,
        id: 11
      },
    ]
  },
  {
    id: 6,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 12
      },
    ]
  },
  {
    id: 7,
    reviews: [
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 13
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Paula Fleri-Soler`,
        date: `December 20, 2016`,
        rating: 8.9,
        id: 14
      },
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Bill Goodykoon`,
        date: `November 18, 2015`,
        rating: 8.9,
        id: 15
      },
    ]
  },
  {
    id: 8,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 16
      },
    ]
  }
];
