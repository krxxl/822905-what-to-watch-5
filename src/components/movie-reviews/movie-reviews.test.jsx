import React from 'react';
import renderer from 'react-test-renderer';
import MovieReviews from './movie-reviews';
import reviews from '../../mocks/reviews';

describe(`Render MovieReviews`, () => {
  it(`Render MovieReviews`, () => {
    const tree = renderer.create(<MovieReviews review={reviews[0]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
