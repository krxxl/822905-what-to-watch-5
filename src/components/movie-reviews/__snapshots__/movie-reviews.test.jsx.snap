// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieReviews Render MovieReviews 1`] = `
Array [
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Kate Muir
      </cite>
      <time
        className="review__date"
        dateTime="May 8, 2019"
      >
        May 8, 2019
      </time>
    </footer>
  </blockquote>,
  <div
    className="review__rating"
  >
    8.9
  </div>,
]
`;
